Update whats-new.mdx (#3295)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

> ## April 2023

Reviewer: I think we should mention the Grid Fundamentals Part one tutorial in the April section.

Author: @jcfilben Can you point me to that ticket? Is that something that was completed in the previous sprint? If so, this What's New is only until mid April.

<!-- BEGIN: What's New Entries -->

## April 2023

- **Brand Refresh**
  - [New brand refresh is now available on Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=0%3A1&t=ViKCtdnRM2CUkNZI-1)!
  - To view details on how to solve for common Figma problems or to migrate to the new theme, check out the [Figma migration guide](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A4&t=ViKCtdnRM2CUkNZI-1).
  - [Wizard titles](https://design-system.hpe.design/templates/wizard) have gone from bold to medium font weight.
- **Figma Component Updates**

>  - Figma components have been updated to use the correct theme colors for placeholder and disabled text. Previously, the placeholder text was xweak. It has been updated to use the correct color, text-weak to meet accessibility. The following components have been updated:

Reviewer: Looks a little strange to have the "Figma Component Updates" and "Figma components have been updated..." at the same bullet point level
<img width="780" alt="Screen Shot 2023-04-26 at 4 03 50 PM" src="https://user-images.githubusercontent.com/54560994/234713094-91985c3f-05c4-4270-afa9-88b76f2a93e4.png">

Author: I had trouble removing the bullet point so it wouldn't look like this. Can you help with that? :)

    - [CheckboxGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5732%3A78041&t=ViKCtdnRM2CUkNZI-1)
    - [RadioButton](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3520%3A83485&t=ViKCtdnRM2CUkNZI-1)
    - [RadioButtonGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5733%3A92878&t=ViKCtdnRM2CUkNZI-1)
    - [TextInput](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A8&t=ViKCtdnRM2CUkNZI-1)
    - [Layer](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5735%3A85900&t=ViKCtdnRM2CUkNZI-1)  
- **FileInput**
  - We have added the alert icon within [FileInput in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=387%3A1967&t=ViKCtdnRM2CUkNZI-1).
- **Data**
  - [Data Filters now have a tooltip](https://design-system.hpe.design/templates/filtering#filtering-with-selectable-results) when hovering over the filter button.

## March 2023

- **Data**

>   - Refactored [Data on DataTable Customization on site](https://design-system.hpe.design/templates/datatable-customization)

Reviewer: ```suggestion
```

- **Brand Refresh**
  - Brand Audit on Grommet has been completed.
  - Older [Card examples](https://design-system.hpe.design/templates/filtering?q=filterin) are now using a Heading with the correct semantic level.
  - New brand refresh now available through out the site! 
  - To view details on changes made to the new theme, view [V5 release notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v5.0.0).
  - To view details on how to migrate from HPE Theme 4.x to 5.x, please visit the [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-4.x-to-5.x).

## February 2023

- **Brand Refresh**
  - Changed HPE theme to brand Metric font.
- **Learn Page**
  - We added a page at the Design System’s top-level navigation called [learn](https://design-system.hpe.design/learn) which is home to tutorials and guides. 
- **Icon Swapping**
  - We updated our icons in Figma so that when you do an instance swap, the icon’s color will automatically apply the correct color when changing between states.
- **DateInput**

>   - Our [DateInput component in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=444%3A9254&t=VBDzNSHpjrMyywm1-1) is now aligned to Grommet and supports days of the week

Reviewer: ```suggestion
  - Our [DateInput component in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=444%3A9254&t=VBDzNSHpjrMyywm1-1) is now aligned to Grommet and supports days of the week.
```

<!-- BEGIN: What's New Entries -->

## April 2023

- **Brand Refresh**
  - Figma scoping has been completed.
  - Designers began updating theme in Figma.
  - [New brand refresh is now available on Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=0%3A1&t=ViKCtdnRM2CUkNZI-1)!
  - To view details on how to solve for common Figma problems or to migrate to the new theme, check out the [Figma migration guide](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A4&t=ViKCtdnRM2CUkNZI-1).

>   - Designers have been setting up calls with different service teams to partner and strategize on how they may begin to migrate to the new theme.

Reviewer: I don't think we need this update

  - [Wizard titles](https://design-system.hpe.design/templates/wizard) have gone from bold to medium font weight.
- **New Global Header**

>   - The team has been working with the HFS team to prototype the new Global Header.

Reviewer: Not sure we need to include this update

    - [CheckboxGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5732%3A78041&t=ViKCtdnRM2CUkNZI-1)
    - [RadioButton](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3520%3A83485&t=ViKCtdnRM2CUkNZI-1)
    - [RadioButtonGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5733%3A92878&t=ViKCtdnRM2CUkNZI-1)
    - [TextInput](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2%3A8&t=ViKCtdnRM2CUkNZI-1)
    - [Layer](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5735%3A85900&t=ViKCtdnRM2CUkNZI-1)  
- **FileInput**
  - We have added the alert icon within [FileInput in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=387%3A1967&t=ViKCtdnRM2CUkNZI-1).
- **Data**
  - [Data Filters now have a tooltip](https://design-system.hpe.design/templates/filtering#filtering-with-selectable-results) when hovering over the filter button.

## March 2023

- **Data**

>   - Refactored [Data on DataTable Customization on site](https://design-system.hpe.design/templates/datatable-customization)

Reviewer: If someone outside of our team were reading this I think it would be hard to understand what was changed and why they should care. 'Refactored' is a bit vague.
```suggestion
  - Changed [DataTable Customization example](https://design-system.hpe.design/templates/datatable-customization) to use the new Data components.
```

- **DateInput**

>   - Updated the heading to use text on site and Figma

Reviewer: Maybe include a little more detail so the reader knows why they should care about this type of update.
```suggestion
  - Changed month and year to use Text instead of Heading to improve accessibility.
```

- **Tabs**
  - Updated [Tabs styling](https://design-system.hpe.design/components/tabs?q=tabs) based on new brand refresh changes.
- **Page Templates**
  - [List + Table Figma templates](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=5639%3A166208&t=IC28JVzDR8NsAZ30-1) have been completed.
  - [Detail/Object View Figma templates](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=5768%3A218145&t=IC28JVzDR8NsAZ30-1) have been completed.
- **Brand Refresh**
  - Brand Audit on Grommet has been completed.
  - Older [Card examples](https://design-system.hpe.design/templates/filtering?q=filterin) are now using a Heading with the correct semantic level.
  - New brand refresh now available through out the site! 
  - To view details on changes made to the new theme, view [V5 release notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v5.0.0).
  - To view details on how to migrate from HPE Theme 4.x to 5.x, please visit the [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-4.x-to-5.x).

## February 2023

- **Brand Refresh**
  - Changed HPE theme to brand Metric font

>   - Conducted a brand audit on GLCP to identify and track unaligned elements to the new and updated HPE Theme.

Reviewer: ```suggestion
```
Not sure that we need this update, it seems like the people that care about this would already be informed and it doesn't relate to all consumers of the design system.

- **Learn Page**
  - We added a page at the Design System’s top-level navigation called [learn](https://design-system.hpe.design/learn) which is home to tutorials and guides. 
- **Icon Swapping**

>   - We updated our icons so that when you do an instance swap, the icon’s color will automatically apply the correct color when changing between states.

Reviewer: Maybe specify that this relates to Figma?

- **DateInput**
  - Our [DateInput component in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=444%3A9254&t=VBDzNSHpjrMyywm1-1) is now aligned to Grommet and supports days of the week
- **Page Templates**

>   - Testing has been completed.

Reviewer: ```suggestion
```
I don't think we need to include this update

- **Layer**
  - [Layers now have overflow property](https://design-system.hpe.design/templates/filtering?q=filte) to ensure that the content does not exceed the container, making it easier for user to view all filter content.
- **Tabs**
  - Updated [Tabs styling](https://design-system.hpe.design/components/tabs?q=tabs) based on new brand refresh changes.
- **Page Templates**
  - List + Table Figma templates have been completed.
  - Detail/Object View Figma templates have been completed.
- **Brand Refresh**
  - Brand Audit on Grommet has been completed
  - Older [Card examples](https://design-system.hpe.design/templates/filtering?q=filterin) are now using a Heading with the correct semantic level.
  - New brand refresh now available through out the site! 
  - To view details on changes made to the new theme, view [V5 release notes](https://github.com/grommet/grommet-theme-hpe/releases/tag/v5.0.0).
  - To view details on how to migrate from HPE Theme 4.x to 5.x, please visit the [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-4.x-to-5.x).
  - Began scoping out Figma work needed to align to the new HPE Theme.
- **PageHeader + Toolbar**

>   - New guidance and examples of when to use certain buttons at the PageHeader and/or the Toolbar level have been created in Figma and will be published on the site soon!

Author: updated
